### ISSUE TITLE
Copy2RAM

### ISSUE THREAD

**Issue by ShalokShalom** [QubesOS/qubes-issues#1264]
Increase the Speed, Boot time and RAM Usage for the Live Mode:

https://en.wikipedia.org/wiki/List_of_Linux_distributions_that_run_from_RAM

**Comment by marmarek** [comment]
Since we're using standard tool to create that Live, I believe it is
already possible, or very easy to add. But the problem is amount of
RAM required for that. On Qubes we need to start not one system, but
multiple VMs and this already requires quite a lot of RAM.

Furthermore some parts of the Live edition is already running from RAM -
namely all the user VMs are stored entirely from RAM.

-- 
Best Regards,
Marek Marczykowski-Górecki
Invisible Things Lab
A: Because it messes up the order in which people normally read text.
Q: Why is top-posting such a bad thing?

**Comment by ShalokShalom** [comment]
Yes, Wayland will reduce the RAM usage a lot: https://plus.google.com/u/0/communities/116669111554447085724/s/wayland

**Comment by ben-grande** [comment]
There are other issues related to live distro.
